access/views.py

```diff
@@ -175,8 +175,11 @@ def aplus_json(request, course_key):
     if course is None:
         raise Http404()
     data = _copy_fields(course, ["name", "description", "lang", "contact",
-        "assistants", "start", "end", "categories",
-        "numerate_ignoring_modules"])
+        "assistants", "start", "end", "categories", "head_urls",
```

Comment:
Put one on a line, so edits are more clear and then sort them. This cloud of words is unreadable.